zappa/handler.py

```diff
@@ -345,18 +345,18 @@ def handler(self, event, context):
 
         # This is the result of a keep alive, recertify
         # or scheduled event.
-        if event.get('detail-type') == u'Scheduled Event':
+        if 'source' in event:
```

Comment:
Can you explain this change? I feel like I'm missing something here, as I don't see the change as being related to the PR.

Reply:
This was submitted almost 2 years ago, so my memory is hazy.

From what I remember the event that I was looking for was handled incorrectly. I think distinguishing factor was that "source" key was set same as in "Scheduled Event" but the detail-type was different. I think this was the unfinished part where proper handling of different types supposed to be implemented. Previously the code just failed back to responding to http requests.

This PR can probably be closed, for my needs I realized that writing my own handler was actually way simpler than using a generalized module like zappa :)